src/components/Registration/Registration.tsx

```diff
@@ -19,6 +20,10 @@ const Registration: FC<RegistrationProps> = (props) => {
   const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     setPage(page + 1);
+    if(page === 4) {
+      setIsLoggedIn(true);
+      <Redirect to='/matched-routes' />
+    }
```

Comment:
This looks good! Maybe we can create a fake logged in user to demonstrate functionality